scene.go

```diff
@@ -23,16 +23,22 @@ type Scene interface {
 	Type() string
 }
 
```

Comment:
Similarly, the `scenes` variable does not need a variable group as the group only contains a single variable.

```diff
- var (
- 	scenes = make(map[string]*sceneWrapper)
- )
+ var scenes = make(map[string]*sceneWrapper)
```